src/layman/common/micka/__init__.py

```diff
@@ -0,0 +1,9 @@
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class MickaNames:
+    metadata_uuid: str
```

Comment:
metadata_uuid -> metadata_id

or

metadata_uuid -> id

Reply:
Renamed to `id`